observations/obs_converters/MIDAS/MIDAS_to_obs.rst

```diff
@@ -0,0 +1,118 @@
+PROGRAM ``MIDAS_to_obs``
+========================
+
+Overview
+--------
+
+MIDAS netCDF file to DART observation converter
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Alex Chartier (University of Bath, UK) is the point-of-contact for this effort.
+
+   "MIDAS runs in Matlab. The raw observations come from GPS receivers as RINEX files, but we can't use them directly
+   just yet ... Currently, the 'slant' (satellite-to-receiver path) observations are inverted by MIDAS to make vertical,
+   column-integrated 'observations' of plasma density."
+
+Data sources
+------------
+
+| Contact Alex for MIDAS observations.
```

Comment:
This seems pretty hokey ... I googled RINEX GPS and got something that looks promising:
https://cddis.nasa.gov/Data_and_Derived_Products/GNSS/RINEX_Version_3.html

Reply:
googling 'midas matlab'   actually gave something useful.